src/rubicon/objc/api.py

```diff
@@ -815,7 +794,9 @@ def objc_class(self):
     def _associated_attr_key_for_name(name):
         return SEL(f"rubicon.objc.py_attr.{name}")
 
-    def __new__(cls, object_ptr, _name=None, _bases=None, _ns=None):
+    def __new__(
+        cls, object_ptr, _name=None, _bases=None, _ns=None, returned_from_method=b""
```

Comment:
`returned_from_method` should have a leading `_` to indicate it's not part of the public API.

Reply:
Done. Apologies for that, it seems I forgot some of the planned "cleanup" when switching this PR from draft to "ready for review".